components/core/SlateMediaObject.js

```diff
@@ -94,7 +96,9 @@ export default class SlateMediaObject extends React.Component {
 
     // TODO(jim): We will need to revisit this later.
     if (type.startsWith("application/unity")) {
-      return <UnityFrame url={url} />;
+      return (
```

Comment:
Can you put these values in the if guard code? Since nothing else needs it.

Reply:
Sure